### PR TITLE
Unify task runner error handling

### PR DESCRIPTION
**Pull request by sndrs**
- updates the task runner to not branch error handling on environment
- (hopefully) errors will always show everywhere, for every task (which they were meant to do, [but didn't](https://teamcity.gu-web.net/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=27045&state=3702%2C3684#_state=3702,3684&focus=3684))
- adds readme and better comments

### CODE REVIEW

**Comment by sndrs** [comment]
broken CSS imports example: 

https://teamcity.gu-web.net/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=27191#_focus=3646

screen shot 2017-01-03 at 14 03 44

**Comment by sndrs** [comment]
broken JS requires example:

https://teamcity.gu-web.net/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=27193#_focus=3740

screen shot 2017-01-03 at 14 05 42

**Comment by sndrs** [comment]
JS lint error example:

https://teamcity.gu-web.net/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=27194#_focus=144

screen shot 2017-01-03 at 14 12 14

File: static/src/stylesheets/_common.scss [hunk 1]
```diff
@@ -25,3 +25,4 @@
 @import 'module/experiments/contributions';
 @import 'module/experiments/embed';
 @import 'module/experiments/svg';
+p
```
Comment (desbo) [inline static/src/stylesheets/_common.scss]: :motor_scooter:

**Comment by sndrs** [comment]
sass lint error example:

https://teamcity.gu-web.net/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=27195#_focus=296

screen shot 2017-01-03 at 14 29 04

File: static/src/stylesheets/_common.scss (hunk 1 above)
Comment (sndrs) [inline static/src/stylesheets/_common.scss]: oop! yeah was deliberately failing the build to check error handling :)

File: tools/task-runner/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# Task runner
+
+This is lightweight task runner that effectively wraps [listr](https://github.com/SamVerschueren/listr) to provide a UI on tasks, which are defined in `tools/__tasks__/`.
+
+It's intended to exist ‘behind the scenes’, and should you should probably be running the tasks it runs from [`make`](https://github.com/guardian/frontend/blob/master/makefile) or something similar.
```
Comment (SiAdcock) [inline tools/task-runner/README.md]: > should you should probably be running

😖

File: tools/task-runner/README.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            description: 'another subtask',
+            task: () => {
+                // do something...
+            }
+        }
+    ]
+}
+```
+
+If the task is array of other tasks, you can also specify that its tasks should run concurrently:
```
Comment (SiAdcock) [inline tools/task-runner/README.md]: `array` needs a determiner

**Comment by SiAdcock** [comment]
Really simplifies thing, dude. Great docs too 👍

File: tools/task-runner/runner [hunk 4]
```diff
@@ -78,18 +13,20 @@ const chalk = require('chalk');
 const figures = require('figures');
 const uniq = require('lodash.uniq');
 
+// name of the tasks directory
 const tasksIdentifier = '__tasks__';
```
Comment (NataliaLKB) [inline tools/task-runner/runner]: Why not name this `tasksDirectory` , then you don't need the comment as it is self explanatory ?

**Review by NataliaLKB — APPROVED** [review]
Looks good 👍

File: tools/task-runner/runner (hunk 4 above)
Comment (sndrs) [inline tools/task-runner/runner]: indeed! i don't know...

**Pull request merged**